README.md

```diff
@@ -1,27 +1,21 @@
-Beneficial Ownership Data Standard
-==================================
+Beneficial Ownership Data Standard (BODS)
+========================================
 
-This repository contains work in progress for the development of a data specification and standard for publication and consumption of data about Beneficial Ownership.
+[![Documentation Status](https://readthedocs.org/projects/beneficial-ownership-data-standard/badge/?version=v0-2-0)](http://standard.openownership.org/en/v0-2-0/?badge=v0-2-0)
```

Comment:
Is there a reason we can't use `latest` as the version here?

Reply:
No. Good spot - I've updated.